docs/architecture/transactions/procedures.md

```diff
@@ -1,70 +1,75 @@
+| `get_balance`             | `[faucet_id]`| `[balance]`| account, note | Returns the `balance` of a fungible asset associated with a `faucet_id`. Panics if the asset is not a fungible asset. |
+| `has_non_fungible_asset`  | `[ASSET]`   | `[has_asset]`| account, note | Returns a boolean `has_asset` indicating whether the non-fungible asset is present in the vault. Panics if the `ASSET` is a fungible asset.  |
+| `add_asset`               | `[ASSET]`   | `[ASSET']`  | account | Adds the specified asset `ASSET` to the vault. Panics under various conditions. If `ASSET` is a non-fungible asset, then `ASSET'` is the same as `ASSET`. If `ASSET` is a fungible asset, then `ASSET'` is the total fungible asset in the account vault after `ASSET` was added to it. |
+| `remove_asset`            | `[ASSET]`   | `[ASSET]`   | account | Remove the specified `ASSET` from the vault. Panics under various conditions.  |
+| `get_vault_commitment`    | `[]`        | `[COM]`     | account, note | Returns a commitment `COM` to the account vault.  |
 
 ### Note
-To import the note procedures set `use.miden::note` at the beginning of the file. All procedures are restricted to the note context.
+
+To import the note procedures, set `use.miden::note` at the beginning of the file. All procedures are restricted to the note context.
 
 | Procedure name           | Inputs              | Outputs               | Context | Description                                                                                                                         |
 |--------------------------|---------------------|-----------------------|---------|-------------------------------------------------------------------------------------------------------------------------------------|
-| `get_assets`             | `[dest_ptr]`        | `[num_assets, dest_ptr]` | note | <details><summary>View</summary>Writes the assets of the currently executing note into memory starting at the specified address. dest_ptr is the memory address to write the assets. num_assets is the number of assets in the currently executing note.</details> |
-| `get_inputs`             | `[dest_ptr]`        | `[dest_ptr]`            | note | <details><summary>View</summary>Writes the inputs of the currently executed note into memory starting at the specified address. dest_ptr is the memory address to write the inputs.</details> |
-| `get_sender`             | `[]`                | `[sender]`             | note | <details><summary>View</summary>Returns the sender of the note currently being processed. Panics if a note is not being processed. sender is the sender of the note currently being processed.</details> |
+| `get_assets`             | `[dest_ptr]`        | `[num_assets, dest_ptr]` | note | Writes the assets of the currently executing note into memory starting at the specified address `dest_ptr `. is the memory address to write the assets. `num_assets` is the number of assets in the currently executing note. |
```

Comment:
I think this HTML code `<details><summary>View</summary>` allows to collapse and expand the text to make the table less clinched. 

Shouldn't we keep that?

Reply:
no, we don't use that across the board ... normally we would do API docs in a different way - not with tables - maybe even importing them with a plugin directly from code ..

a quick fix for this is to make the far column bulleted lists

Reply:
I did bulleted lists for now but there are better ways but it might mean building a plugin ...